libraries/app/database_api.cpp

```diff
@@ -103,6 +108,31 @@ class database_api_impl : public std::enable_shared_from_this<database_api_impl>
     boost::signals2::scoped_connection _block_applied_connection;
 
     bool _disable_get_block = false;
+
+    template <typename MemberObject, typename Service>
+    std::set<account_name_type> lookup_committee_accounts(const std::string& lower_bound_name, uint32_t limit) const
+    {
+        FC_ASSERT(limit <= 1000);
+
+        auto& committee_service = _db.obtain_service<Service>();
+        auto committee = committee_service.get_committee();
```

Comment:
Make get_committee_by_account() method in committee_service. Inside it retrive index by by_account_name. Sorting members by account name in this way is not understandable.